proposals/0000-hyperdb.md

```diff
@@ -0,0 +1,581 @@
+
+Title: **DEP-0000: HyperDB**
+
+Short Name: `0000-hyperdb`
+
+Type: Standard
+
+Status: Undefined (as of 2018-03-XX)
+
+Github PR: [Draft](https://github.com/datprotocol/DEPs/pull/3)
+
+Authors:
+[Bryan Newbold](https://github.com/bnewbold),
+[Stephen Whitmore](https://github.com/noffle),
+[Mathias Buus](https://github.com/mafintosh)
+
+
+# Summary
+[summary]: #summary
+
+HyperDB is a new abstraction layer providing a general purpose distributed
```

Comment:
Is it "new"? Aint this all new?

Reply:
(Not just nitpicking on prose; I'm concerned "new" will be confusing, both now and later when nothing is comparatively newer.)

Reply:
Good point; i'm still finding the right "voice" to use: tense, "we/I", "should/could/must", "one/you".

I was thinking of adding a CI pass with something like http://proselint.com/; we should probably have at least a pass that validates the markdown, and maybe checks out-links for valid HTTP responses.